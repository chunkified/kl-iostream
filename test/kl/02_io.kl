require kliostream;

operator entry() {
  OFStream oStream('data/02_io.txt');
  report(oStream.is_open());
  oStream.write_line('line 01');
  oStream.write_line('line 02');
  oStream.write_line('line 03');
  oStream.write_line('line 04');
  oStream.close();

  IFStream iStream('data/02_io.txt');
  report(iStream.is_open());
  report(iStream.read_line());
  report(iStream.read_line());
  report(iStream.read_line());
  report(iStream.read_line());
  report(iStream.read_line());
  report(iStream.eof());
  iStream.close();
}