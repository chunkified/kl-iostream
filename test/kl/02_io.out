[FABRIC:MT] true
[FABRIC:MT] true
[FABRIC:MT] line 01
[FABRIC:MT] line 02
[FABRIC:MT] line 03
[FABRIC:MT] line 04
[FABRIC:MT] 
[FABRIC:MT] true
